src/openseadragon.js

```diff
@@ -230,8 +230,10 @@
   * @property {Boolean} [flipped=false]
   *     Initial flip state.
   *
-  * @property {Boolean} [overlayContentFlipped=false]
-  *     Initial overlay content flip state.
+  * @property {Boolean} [overlayFlipReversal=true]
```

Comment:
I think we're getting there, but it still seems confusing. Is it going to reverse flipping the overlay entirely? Maybe it should be `overlayFlipContentReversal` or something. I know that's long, but it's kind of a specialized property so that's okay. I suppose `overlayFlipReadable` might be good... Gets to the point of the feature.

Reply:
Other options could be something like `overlayPreserveLTR`, 'overlayPreserveContentDirection' (`direction` being the name of the css property for this), or the even more verbose `overlayPreserve[...]WhenFlipped`. "Preserve" could also be swapped out for "Maintain" in any of the above.

Reply:
I think `overlayPreserveContentDirection` is probably the best one so far.